client/app/pages/groups/GroupsList.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import { react2angular } from 'react2angular';
+
+import Button from 'antd/lib/button';
+import Modal from 'antd/lib/modal';
+import Tooltip from 'antd/lib/tooltip';
+import { Paginator } from '@/components/Paginator';
+import CreateGroupDialog from '@/components/groups/CreateGroupDialog';
+
+import { wrap as liveItemsList, ControllerType } from '@/components/items-list/ItemsList';
+import { ResourceItemsSource } from '@/components/items-list/classes/ItemsSource';
+import { StateStorage } from '@/components/items-list/classes/StateStorage';
+
+import LoadingState from '@/components/items-list/components/LoadingState';
+import EmptyState from '@/components/items-list/components/EmptyState';
+import ItemsTable, { Columns } from '@/components/items-list/components/ItemsTable';
+
+import { Group } from '@/services/group';
+import settingsMenu from '@/services/settingsMenu';
+import { toastr } from '@/services/ng';
+import { currentUser } from '@/services/auth';
+import navigateTo from '@/services/navigateTo';
+import { routesToAngularRoutes } from '@/lib/utils';
+
+class GroupsList extends React.Component {
+  static propTypes = {
+    controller: ControllerType.isRequired,
+  };
+
+  listColumns = [
+    Columns.custom((text, group) => (
+      <div>
+        <a href={'groups/' + group.id}>{group.name}</a>
+        {(group.type === 'builtin') && <span className="label label-default m-l-10">built-in</span>}
+      </div>
+    ), {
+      field: 'name',
+      width: null,
+    }),
+    Columns.custom((text, group) => (
+      <Button.Group>
+        <Button href={`groups/${group.id}`} onClick={e => e.stopPropagation()}>Members</Button>
+        <Button href={`groups/${group.id}/data_sources`} onClick={e => e.stopPropagation()}>Data Sources</Button>
```

Comment:
Quick thing I found when testing features: This button should only appear to admins in order to avoid this:

![403-datasources](https://user-images.githubusercontent.com/3356951/53085031-cab9be80-34e0-11e9-95ad-6210ae90a645.gif)

Reply:
@gabrieldutra You're right, but IMO it feels broken: non-admin can see other group members, why they cannot see datasources they have access to (but cannot add/delete, of course)? I need to think about it :thinking:

Reply:
Decided to keep it as is (non-admins cannot see group datasources).